todo-app/src/Task.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import TaskText from './TaskText';
+import TaskDate from './TaskDate';
+import './task.css';
+
+export default class Task extends React.Component {
+	
+  constructor(props) {
+    super(props);
+
+    // Events and bind to `this`
+    this.markAsDone = this.markAsDone.bind(this);
+  }
+
+  markAsDone(event) {
+    const checked = event.target.checked;
+    this.props.onDoneChange(checked, this.props.task.id);
+  }
+  
+  render() {
+    const { description ,deadline , done } = this.props.task;
```

Comment:
Watch out for the commas/spaces here. I'd definitely recommend using a linter like eslint to automatically check formatting like this.